docs/observability/telemetry-collection.md

```diff
@@ -0,0 +1,282 @@
+# Telemetry Collection
+
+Firebase Genkit Monitoring leverages Google Cloud metrics, traces, and logs to capture telemetry. This 
+document details which metrics, trace attributes, and logs will be collected and what you can expect for 
+latency, quotas, and cost.
```

Comment:
```suggestion
The Firebase telemetry plugin exports a combination of metrics, traces, and logs to Google Cloud Observability. This document details which metrics, trace attributes, and logs will be collected and what you can expect in terms of latency, quotas, and cost.
```

I'm interesting in David, Alex and your thoughts. But I think we shouldn't interchangeably use Firebase telemetry plugin and Firebase Genkit Monitoring. So I am trying to work out the balance for when we use which one. 

My overall gut feel take, is that these documents should generally refer to the plugin if we are talking about configuration and data collection and Firebase Genkit Monitoring when we are talking about the dashboard.

Reply:
Resolved offline. Changed this to be consistently "Firebase telemetry plugin"